package.json

```diff
@@ -40,6 +40,7 @@
     "chalk": "^4.1.2",
     "inquirer": "^8.2.4",
     "pre-commit": "^1.2.2",
+    "proxyquire": "^2.1.3",
```

Comment:
I saw it has a dependency of Fastify, so I tough it could be welcome here.
https://github.com/fastify/fastify/blob/7e50e5307c9a6593a2e43171912630033aa1aacb/package.json#L189